apps/redi-connect/src/pages/app/me/stepper-card/HorizontalStepper.tsx

```diff
@@ -0,0 +1,100 @@
+import { Step, StepLabel, Stepper, Typography } from '@mui/material'
+import StepConnector, {
+  stepConnectorClasses,
+} from '@mui/material/StepConnector'
+import { styled } from '@mui/material/styles'
+import classnames from 'clsx'
+import { Content, Notification } from 'react-bulma-components'
+import './HorizontalStepper.scss'
+
+const QontoConnector = styled(StepConnector)(() => ({
+  [`&.${stepConnectorClasses.alternativeLabel}`]: {
```

Comment:
Wow! Looks like a ton of CSS and MUI prep and work went into this. On first pass it reads like Greek to me. Let's spend some time Thursday morning with you explaining me this magic.